README.md

```diff
@@ -1 +1,183 @@
-# react-capstone-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>GamerPower Application</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[GamerPower\] ](#-gamerpower-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Demonstration Video ](#-demonstration-video-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [GamerPower] <a name="about-project"></a>
+
+**[GamerPower application]** is the ultimate tool for gamers! With GamerPower, you can easily find, track, and collect the best giveaways in gaming. Whether you're looking for free games, beta keys, free in-game loot, or currency for your favorite games, you won't miss out on a single freebie with GamerPower by your side!
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.gamerpower.com/api/giveaways">Open GamePower API</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Global state using Redux]**
+- **[API driven application]**
+- **[Single Page Application]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Demonstration Video -->
+
+## 🚀 Demonstration Video <a name="live-demo"></a>
+
+- [Demonstration Video](https://www.loom.com/share/0ce4f760e4c446c9b89603bbb8851f31)
```

Comment:
- [ ] [REQUIRED] I like the way you were so calm with your presentation 👍🏽 . However, it is a requirement that you should highlight a piece of the code that you find interesting or proud of making. This will also help me as the reviewer to have good knowledge that you developed the project by yourself. Let's fix this 🥇